processing/src/main/java/org/apache/druid/java/util/common/StringUtils.java

```diff
@@ -349,6 +349,23 @@ public static byte[] toUtf8Nullable(@Nullable final String string)
     return toUtf8(string);
   }
 
+  /**
+   * Converts the given JSON string which uses single quotes for field names and
+   * String values to a standard JSON by replacing all occurrences of a single
+   * quote with double quotes.
+   * <p>
+   * Single-quoted JSON is typically easier to read as can be seen below:
+   * <pre>
+   * final String singleQuotedJson = "{'f1':'value', 'f2':5}";
+   *
+   * final String doubleQuotedJson = "{\"f1\":\"value\", \"f2\":5}";
+   * </pre>
+   */
+  public static String singleQuoteToStandardJson(String singleQuotedJson)
```

Comment:
nit: i wonder if it should really be here since I'm having trouble imagining production use cases for this function, do you have other things in mind?

Reply:
Yeah, it bugged me too. Would you prefer if I moved it to `TestUtils` or just marked this method as `@VisibleForTesting`?

Reply:
I think moving it would be better if you don't mind

Reply:
Absolutely! Moved.